.gitignore

```diff
@@ -5,3 +5,4 @@ phpunit.xml
 composer.phar
 package.tar
 /packages.json
+/.phpunit
```

Comment:
this would not work properly for component builds unless we add it in all components too and make the logic locating phpunit much more complex. It would make things even worse by forbidding to test components against Yaml 3.0 as PHPUnit would force using the 2.x component
The issue we want to solve here is that some optional PHPUnit feature (not used in Symfony) make it require the Yaml component 2.x and we don't want that dependencies to be able to test our Yaml component properly in any version.